lib/routes/github/repos.ts

```diff
@@ -28,20 +32,46 @@ export const route: Route = {
 
 async function handler(ctx) {
     const user = ctx.req.param('user');
-
-    const headers = {};
+    const type = ctx.req.param('type') || 'all';
+    const sort = ctx.req.param('sort') || 'created';
+    const direction = ctx.req.param('direction') || 'desc';
```

Comment:
IMO the direction parameter seems unnecessary since the route can't receive any updates when ones repo count exceeds the default `per_page` of 30 in `asc` direction.